### ISSUE TITLE
Enable harbor registry storage provider disable redirect = true

### ISSUE THREAD

**Issue by stonezdj** [goharbor/harbor#7595]
https://github.com/vmware/harbor-boshrelease/issues/15

harbor.yaml should add an option registry_storage_provider.disable.redirect
default value is true
common/templates/registry/config.yml file as shown below and restarting the harbor job allows pushing images to the registry.
if registry_storage_provider.redirect.disable = true
```
storage:
  cache:
    layerinfo: inmemory
  s3:
    region:  *****
    bucket: **********
    encrypt: true
    secure: true
    chunksize: 5242880
    rootdirectory: /harbor-container-registry
  maintenance:
    uploadpurging:
      enabled: false
  delete:
    enabled: true
  redirect:
    disable: true
```
if false
Remove the option
```
   redirect:
      disable: true
```

**Comment by stonezdj** [comment]
https://github.com/goharbor/harbor/blob/3550e2eb23fe7c8f08d5de561d4888aedfe371c5/make/photon/prepare/templates/registry/config.yml.jinja#L17
It should be disable not disabled in config.yml

**Comment by ninjadq** [comment]
fixed in #7770 #7771

**Comment by ninjadq** [comment]
fixed in #7770 and #7771
